tracer/src/Datadog.Trace/Configuration/ImmutableTracerSettings.cs

```diff
@@ -511,6 +513,11 @@ internal ImmutableTracerSettings(TracerSettings settings, bool unusedParamNotToU
         /// </summary>
         internal bool PeerServiceTagsEnabled { get; }
 
+        /// <summary>
+        /// Gets configuration values for changing service names based on configuration
+        /// </summary>
+        internal IReadOnlyDictionary<string, string> PeerServiceNameMappings => _serviceNameMappings;
```

Comment:
```suggestion
        internal IReadOnlyDictionary<string, string> PeerServiceNameMappings => _peerServiceNameMappings;
```

Reply:
Oops. It proves I forgot UTests in the config as well. 
🙇

Reply:
Fixed in latest commits